### ISSUE TITLE
Incorporate sfm-elk

### ISSUE THREAD

**Issue by kerchner** [gwu-libraries/sfm-ui#283]

**Comment by Tanych** [comment]
## Possible Solution
To support twitter, Weibo or other social media in one elk. It means that the elk in one machine. We might take advantage of the [elasticsearch filter] (https://www.elastic.co/guide/en/elasticsearch/reference/2.3/query-dsl-exists-query.html).

Each social media has a field to identify the data. For example, the twitter has `user_id` and Weibo has `w_user_id`. 
The elasticsearch filter can filter the data belong to different social media. We can add the filter in searchSourceJSON.

## Twitter
```json
 "searchSourceJSON": "{\"index\":\"logstash-*\",\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}},\"filter\": {\"exists\" : { \"field\" : \"user_id\"}}}"
```

## Weibo
```json
 "searchSourceJSON": "{\"index\":\"logstash-*\",\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}},\"filter\": {\"exists\" : { \"field\" : \"w_user_id\"}}}"
```
Both have a different field to distinguish the data. The relevant data can show on different dashboard.
However, Twitter and Weibo both can view all the data in elasticsearch.

**Comment by kerchner** [comment]
Fixed in #308
